router/db-router.js

```diff
@@ -0,0 +1,180 @@
+const express = require("express");
+const db = require("../data/db");
+
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  db.find(req.query)
```

Comment:
`find()` method takes no arguments, so there's no reason to pass `req.query` to it - function is going to return all records from db anyway.